avaframe/out3Plot/outDebugPlots.py

```diff
@@ -2,7 +2,7 @@
 import pathlib
 import copy
 import matplotlib
-matplotlib.use('agg')
+# matplotlib.use('agg')
```

Comment:
Please leave this in :smile:

Reply:
oups
We really need to find a solution here